File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-0.java
class Solution {
    public int[] replaceElements(int[] arr) {
        int n = arr.length;
        int res[] = new int[n];
        for(int i=0;i<n-1;i++){
            int max = func(arr,i+1,n);
            res[i] = max;
        }
        res[n-1] = -1;

        return res;
    }

    public int func(int [] arr,int i,int j){
        int max = -1;
        for(int k=i;k<j;k++){

            if(arr[k]>max){
                max = arr[k];
            }
        }
        return max;
    }
}

File: Data Structures & Algorithms/maximum-ascending-subarray-sum/submission-0.py
class Solution:
    def maxAscendingSum(self, arr: List[int]) -> int:
        res = 0
        sum = arr[0]
        for i in range(1,len(arr)):
            if arr[i]>arr[i-1]:
                sum+=arr[i]
                res = max(sum,res)
            else:
                sum = arr[i]
        
        return res